src/IECoreHoudini/SOP_OpHolder.cpp

```diff
@@ -143,6 +143,8 @@ void SOP_OpHolder::doOperation( IECore::Op *op, const GU_DetailHandle &handle, c
 	}
 	catch( boost::python::error_already_set )
 	{
+		// \todo: libIECoreHoudini should not use python. Determine if
```

Comment:
It looks like the real issue here is that `IECorePython::OpWrapper::doOperation` isn't doing any exception management. It should be catching `error_already_set` itself and then using `ExceptionAlgo::translatePythonException()` to convert it to a regular exception that would be covered by the cases below. This is worth doing whether or not IECoreHoudini retains a dependency on python.

Reply:
Thanks for finding that. I spawned an internal ticket for it.